src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -144,6 +144,7 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
         <AnnouncementMessage message={venue.bannerMessage} />
         {template}
         <ChatSidebar />
+        <VideoChatModal />
```

Comment:
> Unless... is this meant to be for the 'video chat invite' stuff? If so, definitely needs a better name.

Yes, this is the idea. The logic for invites will be inside and this will popup. I will improve the name.

Reply:
Cool, sounds good.